netbox/extras/models.py

```diff
@@ -750,11 +750,18 @@ def delete(self, *args, **kwargs):
     @property
     def size(self):
         """
-        Wrapper around `image.size` to suppress an OSError in case the file is inaccessible.
+        Wrapper around `image.size` to suppress an OSError in case the file is inaccessible. When S3 storage is used
+        ClientError is suppressed instead.
         """
+        from django.conf import settings
+        if settings.MEDIA_STORAGE and settings.MEDIA_STORAGE['BACKEND'] == 'S3':
+            from botocore.exceptions import ClientError as AccessError
+        else:
+            AccessError = OSError
```

Comment:
Although trivial, the reassignment of one exception class to another seems almost certain to frustrate future troubleshooting efforts. Can we split this out into two separate return conditions under `if settings.MEDIA_STORAGE ...` instead?

Reply:
Sure! You want me to change the PR, or are you already ahead of me? :)

Reply:
Never mind, already done